manifests/0000_30_capi-operator_02_roles_operator.yaml

```diff
@@ -7,70 +7,8 @@ metadata:
   name: cluster-capi-operator
 rules:
 - apiGroups:
-  - config.openshift.io
+  - '*'
```

Comment:
Admin permissions are fine for now, but we might need to limit them in the future. For example, RBAC resources can be created by CVO, similar to what we plan to do with credentials requests.